app/lego.py

```diff
@@ -186,6 +195,20 @@ def playMp3(self, filename):
         self.startMp3(filename)
         mp3state = 'PLAYING'
 
+    def playPlaylist(self, playlist_filename, mp3_dir):
+        global mp3state
+        list_mp3_to_play = []
+        spotify.pause()
+
+        mp3list = mp3_dir +'/'+ playlist_filename + '/*.mp3'
+        ##logger.info(mp3list)
+
+        list_mp3_to_play = glob.glob(mp3list)
```

Comment:
Insert these lines next to allow shuffle play:
```
if shuffle:
    list_mp3_to_play = random.shuffle(list_mp3_to_play)
```

Reply:
`if shuffle:
    random.shuffle(list_mp3_to_play)`

otherwise return None